torch_geometric/distributed/local_graph_store.py

```diff
@@ -18,10 +18,38 @@ def __init__(self):
         self._edge_attr: Dict[Tuple, EdgeAttr] = {}
         self._edge_id: Dict[Tuple, Tensor] = {}
 
+        self.num_partitions: int = 1
+        self.partition_idx: int = 0
+        self.node_pb: Union[torch.Tensor, Dict[NodeType, torch.Tensor]] = None
+        r""" node_partition_book:  mapping between node ids and partition idx """
+        self.edge_pb: Union[torch.Tensor, Dict[EdgeType, torch.Tensor]] = None
+        r""" edge_partition_book:  mapping between edge ids and partition idx """
+        self.meta: Optional[Dict[Any, Any]] = None
+        r""" meta information related to partition and graph store info """
+        self.labels: Union[torch.Tensor, Dict[EdgeType, torch.Tensor]] = None
```

Comment:
What is `self.labels` and why is it related to edge type information?

Reply:
normally the dataset will also have the labels y and when we do partition we also can move this part into partition file but also we can put outside. if outside for label we need input this label as argument in NeighborLoader and also for the following sub-funcs like dist_neighbor_sampler, etc